src/util/UserphraseSet.cpp

```diff
@@ -76,3 +76,18 @@ void UserphraseSet::swap(UserphraseSet& set) noexcept
     userphrase_.swap(set.userphrase_);
     dedup_.swap(set.dedup_);
 }
+
+void UserphraseSet::insert_shadow(Userphrase&& userphrase)
```

Comment:
That's because shadow in physics is like the copy of an object and cannot be touched in reality.
I don't know. Maybe this reason is just a nonsense.